src/lib/bootloader/udev_mapping.rb

```diff
@@ -49,6 +48,9 @@ def to_mountby_device(dev)
 
       log.info "#{dev} looked as kernel device name: #{kernel_dev}"
 
+# storage-ng
+# rubocop:disable Style/BlockComments
```

Comment:
All the `# storage-ng` comments are kind of FIXMEs, according to this https://github.com/yast/yast-storage-ng/blob/master/doc/installer-hacks.md#temporarily-commented-code